FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -223,16 +230,29 @@ public static function shopSearchProvider()
                 'findologicDI' => false,
                 'isSearchPage' => false,
                 'isCategoryPage' => true,
+                'isManufacturerPage' => false,
                 'fallbackSearchCookie' => null,
                 'expected' => false
             ],
+            'FINDOLOGIC is active on manufacturer pages' => [
+                'activateFindologic' => true,
+                'shopKey' => 'ABCDABCDABCDABCDABCDABCDABCDABCD',
+                'activateFindologicForCategoryPages' => true,
+                'findologicDI' => false,
+                'isSearchPage' => false,
+                'isCategoryPage' => false,
+                'isManufacturerPage' => true,
+                'fallbackSearchCookie' => null,
+                'expected' => true
+            ],
```

Comment:
This does not fail, when removing the fix, because `$isNoSearchAndCategoryPage` will be true.
I suggest to do `'isCategoryPage' => true`.

Reply:
Thanks, we tested it, with and without fix. Now it works as expected.